project/src/main/webapp/index.html

```diff
@@ -40,6 +40,7 @@
         <div class="page-content" id="content">
           <!-- Your content goes here -->
           <h1>Hello There!</h1>
+          <a href="product-lists.html">check available products in product search database</a>
```

Comment:
Nit: I'd capitalize the C in "check" since this is getting displayed to the user.

Reply:
Actually that page is only to check the information in the product search database. It won't be a part of the end product

Reply:
Ah, I see. Ok, feel free to leave it then.